README.md

```diff
@@ -88,6 +88,8 @@ to Mongoose models to ensure schema validation and any schema defaults are appli
 
 **toObjectConfig**: Configuration to pass to the `toObject` method when converting Mongoose models back to plain items, safe for insertion into the bulk operation. Default value is `{depopulate: true, versionKey: false}`.
 
+**Ordered**: Specifies if the bulk operation should be ordered or unordered. Default value is `true`.
```

Comment:
Can you make that lowercase `ordered` to be consistent with the actual property key.